geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/iterators/BaseAggregatingIterator.scala

```diff
@@ -88,43 +79,3 @@ abstract class BaseAggregatingIterator[T <: AnyRef { def isEmpty: Boolean; def c
   override def deepCopy(env: IteratorEnvironment): SortedKeyValueIterator[Key, Value] =
     throw new NotImplementedError()
 }
-
-trait DeduplicatingScan[T <: AnyRef { def isEmpty: Boolean; def clear(): Unit }] extends AggregatingScan[T] {
```

Comment:
Have we removed the need for this?  Or did it move?

Reply:
Yeah, I removed the de-duplication logic, as it was meant for the non-point z2/z3 indices that I dropped support for.